### ISSUE TITLE
Run instructions do not match implementation for angular app

### ISSUE THREAD

**Issue by holly-cummins** [quarkusio/quarkus-workshops#187]
In https://quarkus.io/quarkus-workshops/super-heroes/#microservices-ui, the workshop instructions tell people to run the pom, and `mvn quarkus:dev`. They still have references to the `maven-frontend-plugin`. That was an awkward discovery in a workshop. :)

I started to update the pom to add back the `maven-frontend-plugin`, but it's non-trvial. Looking at the history, I see removing it was intentional. I think we either need to update the adoc to match the codebase, or go with #159's suggestion of introducing quinoa. That's probably the better option, but it might be worth quickly updating the adoc as a transitional measure?

**Comment by cescoffier** [comment]
Happy to delegate that one to only true Agular expert I know: @agoncal :-D

**Comment by oliv37** [comment]
I don't really see the benefit of creating an angular app just for displaying 2 fighters and a list of fights. 
This workshop is about Quarkus and not Angular, It adds useless complexity (you must install node an npm) and could easily have been replaced by a simple `index.html` page containing a little bit of js to achieve the same result.

This solution is actually implemented in the `event-statistics` module where you just have an `index.html` page to consume and display the statistics.

**Comment by agoncal** [comment]
I need to investigate a bit more on this. The Angular app used to be built from scratch, now we give it totally bundled. So we should not have any `pom.xml` involved. The documentation needs to be updated.

But the issues that I have, is that I can't figure out how we did bundle the Angular app. I'll have a look into this.

**Comment by holly-cummins** [comment]
Mm, interesting. When I ran it I just did `npm run` or something similar and then ran it that way, so I assumed we'd update the instructions to match that. But it sounds like maybe there's a cleaner way to run it as a bundled thing. 

If it's too hard, we could tactically suggest something based on `npm run` and then switch to Quinoa as the long term solution, since that's probably most Quarkus-y.
